test/test_linalg.py

```diff
@@ -6351,12 +6264,10 @@ def test_solve_methods_arg_device(self, device):
             b = torch.randn(3, 1, device=b_device)
             A = torch.randn(3, 3, device=A_device)
 
-            # solve and cholesky_solve goes through generic backend dispatch and hit kernel specific device check first
+            # cholesky_solve goes through generic backend dispatch and hit kernel specific device check first
             # triangular_solve goes through specific backend dispatch (CPU/CUDA) and hit auto-generated device check first
             generic_backend_dispatch_err_str = "Expected b and A to be on the same device"
             specific_backend_dispatch_err_str = "Expected all tensors to be on the same device"
-            with self.assertRaisesRegex(RuntimeError, generic_backend_dispatch_err_str):
```

Comment:
nit: could you leave a small test here (or in test_torch.py) that the error you added is working as expected?

Reply:
Sure, I added the test in https://github.com/pytorch/pytorch/pull/70986/commits/68928e736f23e2d922023ac84f5f2622a4e338ed.